src/store/modules/ADempiere/process/actions.js

```diff
@@ -420,7 +420,7 @@ export default {
               link.href = window.URL.createObjectURL(blob)
               link.download = output.fileName
               // download report file
-              if (!['pdf', 'html'].includes(reportType)) {
+              if (!['csv', 'xlsx', 'xls', 'pdf', 'html'].includes(reportType)) {
```

Comment:
Maybe a constant should be added for the supported formats for the report viewer (html, csv, xlxs, ...) and the ones you can view to download (ps, arxml, ...), supported formats for data export (csv, json, xlsx, txt, xml ...)